### ISSUE TITLE
is the operation right?

### ISSUE THREAD

**Issue by XiaosuWang** [ShannonAI/glyce#14]
https://github.com/ShannonAI/glyce/blob/9e73d67a1db4747eca40f47f81821188aaa7380e/glyce/bin/run_bert_glyce_classifier.py#L214

glyph_loss = loss.mean()
may should be:
glyph_loss = glyph_loss.mean()
?

**Comment by ghost** [comment]
This is a typo error. The error may occur in the process of merging code from multiple repo. Thanks for point out it.
